### PR TITLE
Copy CurrencyIsoCode from Contact to Account.

### PR DESCRIPTION
**Pull request by mattparlane**
Hi there...

I created [an issue](https://powerofus.force.com/_ui/core/userprofile/UserProfilePage?u=0058000000BjCKl&tab=sfdc.ProfilePlatformFeed&fId=0D58000003X5b0M) on the Hub a while back, but while it got some generally positive comments it never saw any action. This is still causing me and my team pain on a regular basis, so I'd love to get it fixed.

Like [last time](https://github.com/SalesforceFoundation/Cumulus/pull/2604), I have looked through the source and had a crack at a fix, but I still don't know how to actually test this, so I haven't, I'm just looking to start a discussion and hopefully this points you in the right direction.

Basically, when a Household Account is automatically created from a Contact creation, I think the currency of the Contact should be copied to the Account. Currently when we create a Contact with (eg) AUD, the Account is created as USD (which is our org-wide default).

Thanks!

### CODE REVIEW

**Comment by coriobriensfdo** [comment]
@mattparlane Thank you for checking this in! I can see that you previously signed our contributors CLA, so we don't need you to sign anything this time around. I'll add it to our backlog, and ask the Engineers to take a look.

**Comment by coriobriensfdo** [comment]
**Lurch: add

**Comment by LurchTheButler** [comment]
Tracking <a href="https://foundation.my.salesforce.com/a2x1E000008LlirQAC" target="_blank">W-031981</a>

**Comment by patrick-yan-sf** [comment]
Closing this PR and linking to new Pull Request that incorporated the suggestion - https://github.com/SalesforceFoundation/Cumulus/pull/4141

**Comment by mrbelvedere** [comment]
Included in beta release 3.151 (Beta 16)

**Comment by mrbelvedere** [comment]
Included in production release 3.151
